docs/BR.md

```diff
@@ -3147,45 +3181,85 @@ The following Certificate Policy identifiers are reserved for use by CAs as an o
-   
-    CRLReason MUST be included in the `reasonCode` extension of the CRL entry corresponding to a Subscriber Certificate that is revoked after July 15, 2023, unless the CRLReason is "unspecified (0)". Revocation reason code entries for Subscriber Certificates revoked prior to July 15, 2023, do NOT need to be added or changed.
+Table: revokedCertificates Component
 
-Only the following CRLReasons MAY be present in the CRL `reasonCode` extension for Subscriber Certifificates:
+| __Component__                     | __Presence__    | __Description__ |
+| ----                              | -               | ----- |
+| `serialNumber`                    | MUST            | MUST be byte-for-byte identical to the serialNumber contained in the revoked certificate. |
+| `revocationDate`                  | MUST            | The date and time which revocation occurred. UTCTime (YYMMDDHHMMSSZ) MUST be used for dates up to and including 2049. GeneralizedTime (YYYYMMDDHHMMSSZ) MUST be used for dates after 2049.  | 
+| `crlEntryExtensions`              | *               | See the "crlEntryExtensions Component" table for additional requirements. |
 
-  * **keyCompromise (RFC 5280 CRLReason #1):** Indicates that it is known or suspected that the Subscriber’s Private Key has been compromised; 
-  * **affiliationChanged (RFC 5280 CRLReason #3):** Indicates that the Subject's name or other Subject Identity Information in the Certificate has changed, but there is no cause to suspect that the Certificate's Private Key has been compromised;
-  * **superseded (RFC 5280 CRLReason #4):** Indicates that the Certificate is being replaced because: the Subscriber has requested a new Certificate, the CA has reasonable evidence that the validation of domain authorization or control for any fully‐qualified domain name or IP address in the Certificate should not be relied upon, or the CA has revoked the Certificate for compliance reasons such as the Certificate does not comply with these Baseline Requirements or the CA's CP or CPS;
-  * **cessationOfOperation (RFC 5280 CRLReason #5):** Indicates that the website with the Certificate is shut down prior to the expiration of the Certificate, or if the Subscriber no longer owns or controls the Domain Name in the Certificate prior to the expiration of the Certificate; or
-  * **privilegeWithdrawn (RFC 5280 CRLReason #9):** Indicates that there has been a subscriber-side infraction that has not resulted in keyCompromise, such as the Certificate Subscriber provided misleading information in their Certificate Request or has not upheld their material obligations under the Subscriber Agreement or Terms of Use.
+**Note:** Backdating the revocationDate field is an exception to best practice described in RFC 5280 (Section 5.3.2); however, these requirements specify the use of the revocationDate field to support TLS implementations that process the revocationDate field as the date when the Certificate is first considered to be compromised.
```

Comment:
Also, I'm curious: Do there really exist "TLS implementations that process the revocationDate field as the date when the Certificate is first considered to be compromised"?  Or has this language accidentally been imported from the Code Signing BRs, which permit the RFC5280-non-compliant backdating of the revocationDate field as a workaround for the fact that Microsoft's stack doesn't support the RFC5280 [Invalidity Date](https://www.rfc-editor.org/rfc/rfc5280.html#section-5.3.2) extension?

Reply:
ISTM that TLS implementations only care about the revocation status of a certificate and pay no heed to any timestamps that indicate how far in the past the revocation occurred, whereas Code Signing does care about the "revocation time" and "invalidity date".

Reply:
Thanks for this comment, @robstradling.

I attempted to address the backdating concerns [here](https://github.com/ryancdickson/staging/pull/5/files).

Rather than accepting your suggestion as written (i.e., move the complete paragraph beginning with "When a CA obtains..." to the beginning of the note) - I split the paragraph such that discussion re: CRLReasons follows the CRLReason table. Otherwise, we're introducing language related to CRLReasons that are not yet enumerated. 

Any concerns with how I handled this?

Reply:
Re: your other comments - I agree with you re: "TLS implementations only care about the revocation status of a certificate and pay no heed to any timestamps that indicate how far in the past the revocation occurred".

It's not clear to me the motivation of the existing language re: backdating. @BenWilson-Mozilla - as the author of SC-61, can you offer any background?

Reply:
@ryancdickson That change looks good to me.  Thanks.

(I still think that the BRs should not be permitting the backdating of revocationDate, but tackling that existing language might be considered scope creep for this particular ballot).

Reply:
Thanks for confirming, Rob!

I agree with your comment re: "scope creep," but will take action to create a separate issue such that we can explore this further in a future cleanup ballot.

Reply:
Given it seems unlikely we'll get through the Ballot in 3 rounds of discussion due to some recent comments, I might explore this in a Version 4 (hopefully the last). 

I'll follow-up with @BenWilson-Mozilla to understand why the backdating discussion was pulled into the TLS BRs via SC-61. 

My preference would be to stipulate the revocation date listed on the CRL must be the _actual_ date of revocation. Any opinion on that @robstradling?

While studying the CRLs disclosed to CCADB, we noted a small number of instances where:
- Certificates appeared to be revoked before they were issued  ([example](https://crt.sh/?q=4e48fab0934f50cd3a42d00653089f1a90d56f6157cad82d9afce2e6dc81be4b))
- Certificates appeared to be revoked at the same time as when they were issued ([example](https://crt.sh/?id=7085490113))

As an outside observer, it's impossible to know the _actual_ revocation date - however we can gain some context  by observing when an entry lands on the CRL (not easily monitored using existing public tools).

Reply:
Update: Discussed a bit with @CBonnell and @wthayer during lunch at the F2F. 

In cases where a pre-certificate is signed and the corresponding certificate is _never_ issued, setting revocationDate to equal the notBefore date makes sense. 

I'd still be interested in learning whether there are other valid use cases for backdating revocationDate.

Reply:
My understanding about the language re: backdating is that it was solely related to situations where the revocation reason code was being changed to keyCompromise from some other reason - to say that was allowed. Does that clarification help us navigate a path forward on this issue?

Reply:
Confirming Ben's understanding: The paragraph this comment thread is on is copied from [here](https://www.mozilla.org/en-US/about/governance/policies/security-group/certs/policy/#6-revocation:~:text=Note%3A%20Backdating%20the%20revocationDate%20field%20is%20an%20exception%20to%20best%20practice%20described%20in%20RFC%205280%20(section%205.3.2)) in the MRSP. That version contains a hyperlink to [here](https://wiki.mozilla.org/CA/Revocation_Reasons#Updating_CRL_Entries) which provides some clarifying language.